service/src/main/java/bio/terra/pipelines/service/PipelinesService.java

```diff
@@ -1,10 +1,15 @@
 package bio.terra.pipelines.service;
 
+import bio.terra.common.exception.NotFoundException;
+import bio.terra.common.exception.ValidationException;
+import bio.terra.pipelines.common.utils.PipelineInputTypesEnum;
 import bio.terra.pipelines.common.utils.PipelinesEnum;
 import bio.terra.pipelines.db.entities.Pipeline;
+import bio.terra.pipelines.db.entities.PipelineInputDefinition;
 import bio.terra.pipelines.db.repositories.PipelinesRepository;
-import java.util.List;
-import java.util.UUID;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.*;
```

Comment:
I think generally Terra services avoid `import *` statements.  You can configure IntelliJ to not do that